datafusion/datasource-parquet/src/row_filter.rs

```diff
@@ -649,73 +649,45 @@ mod test {
 
     #[test]
     fn nested_data_structures_prevent_pushdown() {
-        let table_schema = get_basic_table_schema();
-
-        let file_schema = Schema::new(vec![Field::new(
-            "list_col",
-            DataType::Struct(Fields::empty()),
-            true,
-        )]);
```

Comment:
This test was wrong! It wanted to test that `list_col` prevents pushdown because it's a nested type. Instead it was prevented because `list_col` is not in the table / schema!